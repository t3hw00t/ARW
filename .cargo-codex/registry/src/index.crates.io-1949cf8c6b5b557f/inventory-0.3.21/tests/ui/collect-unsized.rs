pub struct Unsized(str);

inventory::collect!(Unsized);

fn main() {}
