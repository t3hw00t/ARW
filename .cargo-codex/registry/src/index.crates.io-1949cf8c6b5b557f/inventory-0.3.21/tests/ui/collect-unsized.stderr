error[E0277]: the size for values of type `str` cannot be known at compilation time
 --> tests/ui/collect-unsized.rs:3:21
  |
3 | inventory::collect!(Unsized);
  |                     ^^^^^^^ doesn't have a size known at compile-time
  |
  = help: within `Unsized`, the trait `Sized` is not implemented for `str`
note: required because it appears within the type `Unsized`
 --> tests/ui/collect-unsized.rs:1:12
  |
1 | pub struct Unsized(str);
  |            ^^^^^^^
note: required by a bound in `Collect`
 --> src/lib.rs
  |
  | pub trait Collect: Sync + Sized + 'static {
  |                           ^^^^^ required by this bound in `Collect`
